networkx/utils/backends.py

```diff
@@ -64,10 +67,12 @@
 loads the backend and calls the corresponding function on the backend along with the
 additional backend-specific ``backend_kwargs``. After calling the function the networkx
 logger displays the ``DEBUG`` message, if the logging is enabled
-(see :ref:`Introspection <introspect>` below). If no compatible
-backend is found or the function is not implemented by the backend, it raises a
-``NetworkXNotImplemented`` exception. And, if the function mutates the input graph or
-returns a graph, graph generator or loader then it tries to convert and run the
+(see :ref:`Introspection <introspect>` below). If no compatible backend is found
+or the function is not implemented by the backend, it will raise ``NotImplementedError``
+unless ``nx.config.fallback_to_nx`` is set to True (default is False), in which case
+it will fall back to run with the default networkx implementation.
+And, if the function mutates the input graph
```

Comment:
To avoid confusion about the term "fall back", perhaps we could state that conversion happens.(?)
```suggestion
it will convert the input graphs to NetworkX classes and run the default
networkx implementation. And, if the function mutates the input graph
```

Reply:
That's better, thanks!